lib/plain/wrappers/wrap.ts

```diff
@@ -82,3 +83,40 @@ export const wrap = <ET extends keyof MRActions, Action extends keyof MRActions[
       headers,
     })
 }
+
+/**
+ * @private
+ */
+export const plainWrap = <ET extends keyof PlainClientAPI, Action extends keyof PlainClientAPI[ET]>(
```

Comment:
Could you please help me understand this change and the necessity of bringing it here?

Reply:
As the typings are right now they are very coupled between the plain client and the common types, you have to duplicate your types in both these places to get it to work, the idea with this change is to only have to modify the `PlainClientAPI` to get it to work, I will consult this with @andipaetzold